public/js/main.js

```diff
@@ -113,13 +121,57 @@
             });
         }
 
+        function displayMessageNotification(message) {
+            var messageFrom = getUserFromId(message.userid).name;
+            toastr.info(message.text, "Message from " + messageFrom);
+        }
+
+        function setIfConversationIsCleared(conversationId, cleared) {
```

Comment:
Rename to setClearedForConversationMessages, or similar. You might be able to think of a better name than that?